src/backends/torch/torchlib.cc

```diff
@@ -346,9 +366,11 @@ namespace dd
       }
 
     // Create the model
-    if (this->_mlmodel._traced.empty() && this->_mlmodel._proto.empty())
-      throw MLLibInternalException(
-          "Use of libtorch backend needs either traced net or protofile");
+    if (this->_mlmodel._traced.empty() && this->_mlmodel._proto.empty()
```

Comment:
I suggest we wrap this long condition into a function that should be easier to maintain in the future. Also, we may want to put a nice comment on why every condition is there. We've seen this growing uncontrolled before in the Caffe lib API !

Reply:
Instead of documentation I suggest using explicit variable name like:
```
bool  unsupported_model_configuration = this->_mlmodel._traced.empty() && this->_mlmodel._proto.empty()
```

Reply:
I will propose a mix of both , let me know after my next push